### ISSUE TITLE
ELI5 license

### ISSUE THREAD

**Issue by agauniyal** [christiannaths/Redacted-Font#30]
Can I use it in a non-commercial open source project if I keep the license files along source and provide attribution in readme? anything more or less than this?

**Comment by christiannaths** [comment]
Licence is here: https://github.com/christiannaths/Redacted-Font/blob/master/OFL.txt

**Comment by agauniyal** [comment]
>The fonts, including any derivative works, can be bundled, embedded, redistributed and/or sold with any software provided that any reserved names are not used by derivative works. 

Understood.

>The fonts and derivatives, however, cannot be released under any other type of license.

Does this means I cannot include them if my work uses MIT license? Or does it means that I have to include it's license along with source as I've stated above. Sorry for bugging, but I want to make sure there isn't any problem with the usage of fonts :smile:

**Comment by christiannaths** [comment]
When you're shipping your software (all other things being equal), you're actually distributing two things: 1) Your software, and; 2) Redacted Font. Each should include its own license.  Your software _includes_ Redacted, which has it's own license.

At that point it is up to the users of your software to respect the original license of the font. As long as you make the font license available to your users, then you are absolved of the responsibility of how they use it. Failure to include the original font license might imply that you're re-releasing the font under your software's MIT license.

At least that's my understanding.
